1003-button-design-guidance.md

```diff
@@ -61,6 +61,13 @@ Buttons have built-in loading and disabled states. There are three sub-states wi
 - Applications **should** display a success state upon completing actions, providing clear feedback to users that their requests were processed successfully.
 - Applications **should** display loading, error, and success (states) that prevent subsequent interactions, avoiding confusion and informing users of ongoing or completed processes.
 
+### Disabled Buttons
+
+- Disabled buttons **should** not have tooltips, as they are inactive and excluded from tab indexes, making tooltips inaccessible to both mouse and keyboard users.
```

Comment:
```suggestion
- Disabled buttons **must** not trigger a tooltip because they are inactive and excluded from tab indexes making tooltips inaccessible to both mouse and keyboard users. A separate button (possibly with an info icon) *may* be placed next to the disabled button to trigger a tooltip.
```

It is not possible for a disabled button, so this is a must not. If acceptable to the design team, a separate tooltip trigger *may* be placed next to the disabled button. cc: @jbusta5